docs/resources/workshop/cd.md

```diff
@@ -28,7 +28,8 @@ Promote an Application using CD with GitOps and ArgoCD
     ```
 
 1. Verify Application is deployed in **QA**
-    - Select ArgoCD from the Console Link and login using OpenShift login
+    - Get ArgoCD's `admin` password by running `oc extract secrets/openshift-gitops-cluster --keys=admin.password -n openshift-gitops --to=-`  
```

Comment:
The password is two different places depending OCP46 or OCP47+

OCP46: `secrets/argocd-cluster-cluster`
OCP47: `secrets/openshift-gitops-cluster`

We are making fixes to have it easier for the user to just run `oc credentials`
WIth OCP48 the OpenShift SSO would be fix and user can just click the "Login with OpenShift"